client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -42,6 +43,9 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 	const { navigate, params } = useFlowNavigation();
 	const currentStepRoute = params.step || '';
 
+	// Start tracking performance for this step.
+	useStartStepperPerformanceTracking( flow.name, currentStepRoute );
```

Comment:
```suggestion
	useStartStepperPerformanceTracking( param.flow , currentStepRoute );
```

Some flow will have variant name

Reply:
If we start the event on here we will miss a lot of setup code.
It is better to start [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/index.tsx#L90)

Reply:
> If we start the event on here we will miss a lot of setup code. It is better to start [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/index.tsx#L90)

I'm not sure I follow. I already added code to the top of `App.boot` in `client/landing/stepper/index.tsx` to handle the full page load case. This one here is only meant to handle in-app navigations, not initial load. Would a comment help?

Reply:
Sorry, the correct comment is [here](https://github.com/Automattic/wp-calypso/pull/92130#discussion_r1658484797).